src/Altinn.Platform/Altinn.Platform.Storage/Storage/appsettings.json

```diff
@@ -20,7 +20,8 @@
     "OpenIdWellKnownEndpoint": "http://localhost:5040/authentication/api/v1/openid/",
     "Hostname": "at22.altinn.cloud",
     "RuntimeCookieName": "AltinnStudioRuntime",
-    "BridgeApiAuthorizationEndpoint": "http://localhost:5055/sblbridge/authorization/api/"
+    "BridgeApiAuthorizationEndpoint": "http://localhost:5055/sblbridge/authorization/api/",
+    "CacheResourceLifeTimeInSeconds":  600
```

Comment:
Any input on how long the app texts should be cached for? Should the setting be solely for text resources?

Reply:
fixed